_topic_map.yml

```diff
@@ -1275,23 +1275,23 @@ Name: Monitoring
 Dir: monitoring
 Distros: openshift-enterprise,openshift-webscale,openshift-origin
 Topics:
-- Name: Cluster monitoring
-  Dir: cluster_monitoring
-  Topics:
-  - Name: About cluster monitoring
-    File: about-cluster-monitoring
-  - Name: Configuring the monitoring stack
-    File: configuring-the-monitoring-stack
-  - Name: Managing cluster alerts
-    File: managing-cluster-alerts
-  - Name: Examining cluster metrics
-    File: examining-cluster-metrics
-  - Name: Accessing Prometheus, Alertmanager, and Grafana
-    File: prometheus-alertmanager-and-grafana
-- Name: Monitoring your own services
-  File: monitoring-your-own-services
+- Name: Understanding OpenShift monitoring
+  File: understanding-openshift-monitoring
+- Name: Configuring platform monitoring
+  File: configuring-platform-monitoring
+- Name: Managing metrics
```

Comment:
Not sure if managing metrics is very indicative to me that is about custom user metrics. If we already say platform monitoring, why not call it user or application monitoring here?

Same question goes for other user workload monitoring sections.

Reply:
@lilic thank you for highlighting this. I have now updated the title of that one assembly, plus the titles for a number of the modules (sub-sections) that relate specifically to user workload monitoring. I think it is a lot clearer now throughout the titles.

Please let me know if you are happy with the changes.

Reply:
Disagree on @lilic point. "Application Monitoring" or anything else would really lean too much towards “developer apps” rather than focusing on the general “you can monitor everything” approach, IMHO. Again “User Workload Monitoring” is our actual tenant-based monitoring solution for people who have additional targets run either on top or below OpenShift and are not covered by our turn-key monitoring experience.

Reply:
I have now changed this title to 'Managing metrics for your own workloads'.